pages/content/cookies.mdx

```diff
@@ -2,18 +2,18 @@
 title: "Cookies"
 ---
 
-<div class="govuk-notification-banner govuk-notification-banner--success" data-cookie-confirmation="true" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
-  <div class="govuk-notification-banner__header">
-    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+<div className="govuk-notification-banner govuk-notification-banner--success" data-cookie-confirmation="true" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div className="govuk-notification-banner__header">
+    <h2 className="govuk-notification-banner__title" id="govuk-notification-banner-title">
       Success
     </h2>
   </div>
-  <div class="govuk-notification-banner__content">
-    <p class="govuk-notification-banner__heading">
+  <div className="govuk-notification-banner__content">
+    <p className="govuk-notification-banner__heading">
       Your cookie settings were saved
     </p>
-    <p class="govuk-body">
-      <a class="govuk_link previous-page govuk-!-margin-top-1" href="#">
+    <p className="govuk-body">
+      <a className="govuk_link previous-page govuk-!-margin-top-1" href="#">
```

Comment:
I'd be interested to know what the `className` pattern here is for?

Reply:
it's still React so complier will throw an warning. :)

Reply:
aha! cool